src/Web/Slack.hs

```diff
@@ -51,23 +50,15 @@ import Network.HTTP.Client (Manager, newManager)
 -- http-client-tls
 import Network.HTTP.Client.TLS (tlsManagerSettings)
 
--- mtl
-import Control.Monad.Reader
-
 -- servant
 import Servant.API hiding (addHeader)
+import Servant.Client.Core (AuthenticatedRequest)
 
 -- servant-client
 import Servant.Client hiding (Response, baseUrl)
 
-#if MIN_VERSION_servant(0,16,0)
-import Servant.Client.Core (AuthClientData, AuthenticatedRequest, Request, mkAuthenticatedRequest, addHeader)
-#else
-import Servant.Client.Core.Internal.Auth
-import Servant.Client.Core (Request, addHeader)
-#endif
-
```

Comment:
I dropped the old versions since I don't sincerely believe that we are likely to correctly support them.